Assets/Scripts/UI/Pickup Pup/ShelterScreen/PPShelterUIController.cs

```diff
@@ -0,0 +1,37 @@
+﻿/*
+ * Author: Isaiah Mann
+ * Description: Controls the shelter screen
+ */
+
+public class PPShelterUIController : PPUIController
+{
+    DogSlot[] availableDogPortraits;
+    DogDatabase database;
+
+    #region MonoBehaviourExtended Overrides
+
+    protected override void setReferences()
+    {
+        base.setReferences();
+        availableDogPortraits = GetComponentsInChildren<DogSlot>();
+    }
+
+    protected override void fetchReferences()
+    {
+        base.fetchReferences();
+        database = DogDatabase.GetInstance;
+        populateAvailableDogs(database);
+    }
+
+    #endregion
+
+    void populateAvailableDogs(DogDatabase database)
```

Comment:
@gracebarsny Can you override to use a random seed that is somehow generated by the realtime day: 
- Random Seed: https://msdn.microsoft.com/en-us/library/ctssatww(v=vs.110).aspx
- Realtime Day: https://msdn.microsoft.com/en-us/library/system.datetime.day(v=vs.110).aspx

I can see if I can add a quick PR w/ a subclass of RandomBuffer containing the above functionality